format/format.go

```diff
@@ -109,6 +109,9 @@ const (
 	OPUS_PACKET         = "opus_packet"
 	PCAP                = "pcap"
 	PCAPNG              = "pcapng"
+	PG_BTREE            = "pg_btree"
+	PG_CONTROL          = "pg_control"
+	PG_HEAP             = "pg_heap"
```

Comment:
Hi, i did quite large refactor how formats are registered to allow some new things in the futue (probe via schema, filename etc). To make it build again change to:
```go
	PgBtree            = &decode.Group{Name: "pg_btree"}
	PgControl          = &decode.Group{Name: "pg_control"}
	PgHeap             = &decode.Group{Name: "pg_heap"}
```

Reply:
It's done